layer.md

```diff
@@ -44,10 +44,45 @@ Where supported, MUST include file attributes for Additions and Modifications in
     * Group Name (`gname`) *secondary to `gid`*
 * Mode (`mode`)
 * Extended Attributes (`xattrs`)
-* Symlink reference (`linkname`)
+* Symlink reference (`linkname` + symbolic link type)
+* [Hardlink](#hardlinks) reference (`linkname`)
 
 [Sparse files](https://en.wikipedia.org/wiki/Sparse_file) SHOULD NOT be used because they lack consistent support across tar implementations.
 
+#### Hardlinks
+
+Hardlinks are a [POSIX concept](http://pubs.opengroup.org/onlinepubs/9699919799/functions/link.html) for having one or more directory entries for the same file on the same device.
```

Comment:
On Thu, Sep 22, 2016 at 02:14:30PM -0700, Stephen Day wrote:
> @vbatts I still think that condition needs to be added to the clause
> above. The issue is that two files with the same inode but different
> devices ids are two separate hardlinks.

He covers that [in Markdown in the next paragraph][1].  Are you just asking for “file” → “inode” here?

[1]: https://github.com/opencontainers/image-spec/pull/336/files#diff-dc2bb3b4eddcffe96e682193926e8ff6R59